src/test/java/com/egov/core-services/tests/zuul.feature

```diff
@@ -0,0 +1,109 @@
+        * def OccupancyType = mdmsStatePropertyTax.OccupancyType[1].code
+        * def UsageCategory = mdmsStatePropertyTax.UsageCategory[0].code
+        * def builtUpArea = 2000
+        * def UsageCategoryMajor = mdmsStatePropertyTax.UsageCategoryMajor[0].code
+        * def landArea = 800
+        * def PropertyType = mdmsStatePropertyTax.PropertyType[3].code
+        * def noOfFloors = 1
+        * def OwnerShipCategory = mdmsStatePropertyTax.OwnerShipCategory[3].code
+        * def name = randomString(10)
+        * def mobileNumber = '78' + randomMobileNumGen(8)
+        * def fatherOrHusbandName = randomString(10)
+        * def relationship = commonConstants.parameters.relationship[randomNumber(commonConstants.parameters.relationship.length)]
+        * def OwnerType = mdmsStatePropertyTax.OwnerType[5].code
+        * def gender = commonConstants.parameters.gender[randomNumber(commonConstants.parameters.gender.length)]
+        * def isCorrespondenceAddress = true
+        * def source = commonConstants.parameters.source
+        * def channel = commonConstants.parameters.channel
+        * def addressProofDocumentType0 = mdmsStatePropertyTax.Documents[0].dropdownData[0].code
+        * def addressProofDocumentType1 = mdmsStatePropertyTax.Documents[1].dropdownData[0].code
+        * def addressProofDocumentType2 = mdmsStatePropertyTax.Documents[2].dropdownData[0].code
+        * def addressProofDocumentType3 = mdmsStatePropertyTax.Documents[3].dropdownData[0].code
+        * def addressProofDocumentType4 = mdmsStatePropertyTax.Documents[4].dropdownData[0].code
+        * def creationReason = commonConstants.parameters.creationReason
+        * def businessService = mdmsStatePropertyTax.PTWorkflow[1].businessService
+        * def propertyServicesConstants = read('../../municipal-services/constants/propertyServices.yaml')
+    
+    @zuul_01 @zuul @regression @positive @smoke
+        Scenario: Test to create a employee
+    * call read('../../business-services/pretest/egovHrmsPretest.feature@createEmployeeSuccessfully')
+    * def code = hrmsResponseBody.Employees[0].user.userName
+    * assert hrmsResponseBody.ResponseInfo.status == commonConstants.expectedStatus.success
+    * assert hrmsResponseBody.Employees[0].user.name == name
+    * assert hrmsResponseBody.Employees[0].user.mobileNumber == mobileNumber
+
+    @zuul_02 @zuul  @positive @regression  @municipalServices
+    Scenario: Create Property with valid payload
+    # Create a property
+    * call read('../../municipal-services/pretests/propertyServicesPretest.feature@createPropertySuccessfully')
+    # Validate response body
+    * match propertyServiceResponseBody.Properties[0].id == "#present"
+    * match propertyServiceResponseBody.Properties[0].propertyId == "#present"
+    * match propertyServiceResponseBody.Properties[0].status == "INWORKFLOW"
+    * match propertyServiceResponseBody.Properties[0].tenantId == tenantId
+
+    @zuul_05  @zuul @regression @negative
```

Comment:
please add two more tags here. i.e. `@coreServices`, and also specify whether this is a `positive` or `negative test. Like, `@positive` or `@nagetive`.